tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/string/StringTest.java

```diff
@@ -481,14 +481,14 @@ public void testSplit() {
     }
 
     @Test(expectedExceptions = {BLangRuntimeException.class},
-            expectedExceptionsMessageRegExp = "error: error, message: nullReferenceException.*")
+            expectedExceptionsMessageRegExp = "error: NullReferenceException.*")
     public void testSplitNull() {
         BValue[] args = {new BString(null), new BString(" ")};
         BRunUtil.invoke(result, "split", args);
     }
 
     @Test(expectedExceptions = {BLangRuntimeException.class},
-            expectedExceptionsMessageRegExp = "error: error, message: nullReferenceException.*")
+            expectedExceptionsMessageRegExp = "error: NullReferenceException.*")
```

Comment:
It would be nice to have a constant for this `"error: NullReferenceException.*"` then we might have to change only in one place if there are new changes.